assem16399/Diva
# feat(ProductDetails) 🏗 Build ProductDetails Data Layer

## Jira Ticket: [DIVA-30](https://aasemhany.atlassian.net/browse/DIVA-30)

## Description 📖 

Implementing ProductDetails Feature data layer (Models, ApiService, ApiRepo)


## How to Test 📋 
check the ProductDetails Feature Data Layer folder


[DIVA-30]: https://aasemhany.atlassian.net/browse/DIVA-30?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
